recipe/meta.yaml

```diff
@@ -13,12 +13,12 @@ source:
 
 build:
   number: 0
-  script: pip install --no-deps .
```

Comment:
There are a few feedstocks that only do the right thing about its `entrypoints` when pip installed. See the `nbgrader` [issue](https://github.com/conda-forge/nbgrader-feedstock/pull/13) as an example. The same happens with other `jupyter` related feedstocks and that is why @minrk added `pip` here. (Although this particular feedstock does not suffer from that problem.)

@msarahan and I can had a brief chat about this at SciPy but nothing is "formalized" yet. We basically need to know what means for `conda-build` the change from `python setup.py install` to `pip install --no-deps .` (Actually I found out that we need to use `python -m pip install --no-deps .` in some cases, not sure why :-/)